test/Interop/Cxx/foreign-reference/pass-as-parameter.swift

```diff
@@ -0,0 +1,54 @@
+// RUN: %target-run-simple-swift(-I %S/Inputs -cxx-interoperability-mode=upcoming-swift -Xfrontend -disable-availability-checking)
```

Comment:
@egorzhdan, all `%target-run-simple-swift` tests have to include `// REQUIRES: executable_test`, else they [fail in host-test mode](https://github.com/swift-android-sdk/swift-docker/actions/runs/17189997147/job/48764301769).